integration/test/config-test.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+/* eslint max-len: "off" */
+/* eslint no-return-assign: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const {version} = require('../package.json');
+
+const TestUtil = require('../test-util');
+const util = new TestUtil();
+
+describe('Configuration', function() {
+  this.timeout(5000);
+
+  before(async () => {
+    await util.open();
+  });
+
+  after(async () => {
+    await util.close();
+  });
+
+  describe.only('--agent', function () {
+    it('should have default user agent', async () => {
+      await util.generate(1);
+      await util.waitForSync();
+
+      const peer = util.node.pool.peers.head();
+      assert.strictEqual(peer.agent, '/hnsd:1.0.0/');
```

Comment:
Does the test need to be updated with every version bump? Could pick from package.json (which I think was the idea for adding version there)

Edit: yeah just saw version used below

Reply:
yep! fixing...